Library/Formula/bmdtools.rb

```diff
@@ -0,0 +1,19 @@
+require 'formula'
```

Comment:
fixed in https://github.com/Homebrew/homebrew/commit/f46545af73339a8bffec14644169e1e652d88d98